src/app/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-table-mobile/ubs-admin-employee-table-mobile.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, DoCheck, Input } from '@angular/core';
+import { Page } from 'src/app/ubs-admin/models/ubs-admin.interface';
+import { UbsAdminEmployeeTableComponent } from '../ubs-admin-employee-table/ubs-admin-employee-table.component';
+
+@Component({
+  selector: 'app-ubs-admin-employee-table-mobile',
+  templateUrl: './ubs-admin-employee-table-mobile.component.html',
+  styleUrls: ['./ubs-admin-employee-table-mobile.component.scss']
+})
+export class UbsAdminEmployeeTableMobileComponent implements DoCheck {
+  @Input()
+  dataSource;
+
+  @Input()
+  screenWidth: number;
+
+  isLoading = true;
+  chooseItemId = 0;
+
+  constructor(private employeeTableFunc: UbsAdminEmployeeTableComponent) {}
```

Comment:
Wait. it's strange that you inject a component inside another component
You shouldn't do this

Reply:
resolve it too.